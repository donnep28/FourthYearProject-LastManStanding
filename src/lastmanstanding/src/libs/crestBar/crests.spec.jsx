import React from 'react';
import {shallow} from 'enzyme';
import Crests from './crests';

describe('crests', () => {
  const props = {
    screenWidth: 1000,
    crests: [
      {
        'Abbreviation': 'ARS',
        'Crest': 'https://crests.football-data.org/57.svg',
        'TeamName': 'Arsenal FC',
        'Website': 'https://www.arsenal.com/',
      },
      {
        'Abbreviation': 'AVL',
        'Crest': 'https://crests.football-data.org/58.svg',
        'TeamName': 'Aston Villa FC',
        'Website': 'https://www.avfc.co.uk/',
      },
    ],
  };
  describe('Snapshot', () => {
    it('renders', () => {
      const component = shallow(
          <Crests {...props}/>,
      );
      expect(component).toMatchSnapshot();
    });
  });
});
