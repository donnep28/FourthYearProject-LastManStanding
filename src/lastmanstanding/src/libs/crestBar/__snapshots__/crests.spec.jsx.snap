// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`crests Snapshot renders 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <div
    className="img-wrapper"
    key="Arsenal FC"
    style={
      Object {
        "padding": "5px",
      }
    }
  >
    <a
      href="https://www.arsenal.com/"
    >
      <img
        alt="team crests"
        className="hover-zoom"
        height="40px"
        src="https://crests.football-data.org/57.svg"
      />
    </a>
  </div>
  <div
    className="img-wrapper"
    key="Aston Villa FC"
    style={
      Object {
        "padding": "5px",
      }
    }
  >
    <a
      href="https://www.avfc.co.uk/"
    >
      <img
        alt="team crests"
        className="hover-zoom"
        height="40px"
        src="https://crests.football-data.org/58.svg"
      />
    </a>
  </div>
</div>
`;
